package-lock.json

```diff
@@ -2010,7 +2010,7 @@
     "add-stream": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/add-stream/-/add-stream-1.0.0.tgz",
-      "integrity": "sha512-qQLMr+8o0WC4FZGQTcJiKBVC59JylcPSrTtk6usvmIDFUOCKegapy1VHQwRbFMOFyb/inzUVqHs+eMYKDM1YeQ==",
```

Comment:
we need lock chages? wonder what's happening here cause ive seen this a few times lately

Reply:
really not sure. ill undo them here

Reply:
npm version maybe?